x-pack/plugins/siem/public/endpoint_alerts/store/selectors.ts

```diff
@@ -47,6 +47,13 @@ export const isOnAlertPage = (state: Immutable<AlertListState>): boolean => {
   return state.location ? state.location.pathname === '/endpoint-alerts' : false;
 };
 
+/**
+ * Returns a boolean based on whether or not the user navigated within the alerts page
+ */
+export const isAlertPageTabChange = (state: Immutable<AlertListState>): boolean => {
+  return isOnAlertPage(state) && !!state.location?.state?.isTabChange;
```

Comment:
:grey_question:  DOUBLE BANG ( !! ) :rofl: I like these and I use them, but that only makes 2 of us. Rest of the team prefers we don't. Should get coerced to a bool anyway here. _I_ think it makes it more readable but :shrug:

Reply:
Funny enough, I had second thoughts about this when I did it, but thought I'd give it a shot 😂. Shot down lol. I do like me some coercion, but typescript did not as I would have to change the return type to `boolean | undefined` since the tab change parameter could be undefined. Just went with an explicit check instead